### ISSUE TITLE
Undefined reference to 'ergodox_led_all_off' and other ErgoDox LED functions

### ISSUE THREAD

**Issue by nstickney** [qmk/qmk_firmware#1374]
A recent commit introduced this problem; I can't seem to pinpoint the cause. `make ergodox-ez-default-teensy`, which is the command I generally use to update my keyboard, throws the errors below (cleaned up for viewing). The functions it is looking for are all in keyboards/ergodox/ez/ez.h; is there some reason it isn't finding them anymore? [Eliminating the LED functions from my keymap, as well as `ez.c` and `matrix.c`](https://github.com/nstickney/qmk_firmware/commit/f397402e101774eedd1748b1e31c7386f927d4ed) fixes the problem, but now my LEDs do nothing (and I can't tell what layer I'm in).

> Linking: .build/ergodox_ez_default.elf                                                              [ERRORS]
 | 
 | .build/obj_ergodox_ez_default/keyboards/ergodox/ez/ez.o: In function 'ergodox_blink_all_leds':
 | ./keyboards/ergodox/ez/ez.c:32: undefined reference to 'ergodox_led_all_off'
 | ./keyboards/ergodox/ez/ez.c:33: undefined reference to 'ergodox_led_all_set'
 | ./keyboards/ergodox/ez/ez.c:34: undefined reference to 'ergodox_right_led_1_on'
 | ./keyboards/ergodox/ez/ez.c:36: undefined reference to 'ergodox_right_led_2_on'
 | ./keyboards/ergodox/ez/ez.c:38: undefined reference to 'ergodox_right_led_3_on'
 | ./keyboards/ergodox/ez/ez.c:40: undefined reference to 'ergodox_right_led_1_off'
 | ./keyboards/ergodox/ez/ez.c:42: undefined reference to 'ergodox_right_led_2_off'
 | ./keyboards/ergodox/ez/ez.c:44: undefined reference to 'ergodox_right_led_3_off'
 | ./keyboards/ergodox/ez/ez.c:47: undefined reference to 'ergodox_led_all_off'
 | .build/obj_ergodox_ez_default/keyboards/ergodox/keymaps/default/keymap.o: In function 'matrix_scan_user':
 | ./keyboards/ergodox/keymaps/default/keymap.c:206: undefined reference to 'ergodox_board_led_off'
 | ./keyboards/ergodox/keymaps/default/keymap.c:207: undefined reference to 'ergodox_right_led_1_off'
 | ./keyboards/ergodox/keymaps/default/keymap.c:208: undefined reference to 'ergodox_right_led_2_off'
 | ./keyboards/ergodox/keymaps/default/keymap.c:209: undefined reference to 'ergodox_right_led_3_off'
 | ./keyboards/ergodox/keymaps/default/keymap.c:213: undefined reference to 'ergodox_right_led_1_on'
 | ./keyboards/ergodox/keymaps/default/keymap.c:216: undefined reference to 'ergodox_right_led_2_on'
 | collect2: error: ld returned 1 exit status
 | 
make[1]: *** [tmk_core/rules.mk:287: .build/ergodox_ez_default.elf] Error 1
Make finished with errors
make: *** [Makefile:508: ergodox-ez-default-teensy] Error 1

**Comment by fredizzimo** [comment]
This should be fixed once https://github.com/qmk/qmk_firmware/pull/1331, is merged in. 

Since the issue is a duplicate of https://github.com/qmk/qmk_firmware/issues/1314, I'm closing this.

**Comment by nstickney** [comment]
Thanks!
